src/main/assembly/dist/bag/metadata/files/2018/files.xsd

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2012 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:dc="http://purl.org/dc/elements/1.1/"
+    xmlns:f="http://easy.dans.knaw.nl/schemas/bag/metadata/files/" targetNamespace="http://easy.dans.knaw.nl/schemas/bag/metadata/files/"
+    elementFormDefault="qualified" attributeFormDefault="unqualified">
+
+    <!-- =================================================================================== -->
+    <xs:annotation>
+        <xs:documentation xml:lang="en"> This schema specifies a metadata-format for describing files and folders in a SIP. An instance of this
```

Comment:
looks like a formatting destroyed the layout of this documentation (compare it to the 2017/09 version). Same for the rest of the document!

Make sure that in `IntelliJ IDEA > Preferences... > Editor > Code Style > XML > Other` the boxes '_Keep line breaks_' and '_Keep line breaks in text_' are checked!